hepcrawl/spiders/wsp_spider.py

```diff
@@ -89,16 +89,24 @@ def __init__(self, package_path=None, ftp_folder="WSP", ftp_host=None, ftp_netrc
     def start_requests(self):
         """List selected folder on remote FTP and yield new zip files."""
         if self.package_path:
-            yield Request(self.package_path, callback=self.handle_package_file)
+            dummy, new_files = local_list_files(
+                self.package_path,
+                self.target_folder
+            )
+
+            for _file in new_files:
```

Comment:
Instead of `_file` and `new_files`, it's better to use `file_path` and `new_files_paths` so you know it's a string representing the file path, and not a file object (that is usually clear when you write the code yourself, but for someone else it might become very confusing when debugging).